rir/src/runtime/Code.h

```diff
@@ -89,13 +92,18 @@ struct Code {
      *   ---------------------------------------------------------------------
      *   code stream   BC                pad4(codeSize)
      *
-     *   skiplist      (instr offset,    2 * skiplistLength * sizeof(unsigned)
-     *                  src index)
-     *
-     *   srcList       cp_idx (ast)      srcLength * sizeof(unsigned)
+     *   srcList       cp_idx (ast)      srcLength * sizeof(SrcEntry)
      *
      */
 
+    // The source list contains pcOffset to src index
+    struct SrclistEntry {
+        unsigned pcOffset;
+        unsigned srcIdx;
+    };
+    static_assert(sizeof(SrclistEntry) == 2 * sizeof(unsigned),
+                  "Needs to correspond to the size of the record_call_ BC");
```

Comment:
Wrong assert message.

Reply:
actually this is not even needed anymore in this version, where we use an actual array, to index the sourcelist entries